src/integrationTest/resources/application.properties

```diff
@@ -2,3 +2,53 @@ env=${test_environment:local}
 TEST_URL=https://localhost:4044
 
 case.orchestration.service.base.uri=#{environment["TEST_URL"].replaceFirst("https://", "http://")}
+case.orchestration.prepopulate.uri=${case.orchestration.service.base.uri}/getSolicitorAndFeeDetails
+
+###############################################
+#  Logging                                    #
+###############################################
+logging.level.uk.gov.hmcts.ccd=DEBUG
+logging.level.org.springframework.web=DEBUG
+
+###############################################
+#  IDAM                                       #
+###############################################
+auth.idam.client.baseUrl=https://idam-api.aat.platform.hmcts.net
+auth.idam.client.redirectUri=https://manage-case.aat.platform.hmcts.net/oauth2/callback
+auth.idam.client.secret=DBssSOq0KKLNBf2z
+auth.idam.client.clientId=xuiwebapp
+
```

Comment:
secrets need to be replaced

Reply:
we will sort this once we have secrets pulled from azure